docs/iot-gateway/config/opc-ua.md

```diff
@@ -196,11 +331,48 @@ This part of configuration will look like:
             "method": "multiply",
             "arguments": [2, 4]
           }
-        ],
+        ]
 ```
 
+Also, every telemetry and attribute parameter has get and set RPC methods out of the box, so you don't need to configure
```

Comment:
Maybe it will be better to highlight GET and SET methods.

Reply:
Done